docs/BigQueryMultiTable-batchsink.md

```diff
@@ -22,6 +22,12 @@ Properties
 ----------
 **Reference Name:** Name used to uniquely identify this sink for lineage, annotating metadata, etc.
 
+**Use Connection:** Whether to use a connection, if a connection is used,
+the credentials does not need to be provided.
+
+**Connection:** Name of the connection to use, should use the macro function ${conn:(connection-name)} to provide.
```

Comment:
I think the macro should be  ${conn(connection-name)} , there is no colon

Reply:
I used this from here, https://github.com/data-integrations/google-cloud/blame/develop/docs/GCSFile-batchsource.md#L31, @yaojiefeng what do you think?

Reply:
Sean is correct, seems like there is a typo in the description. the correct one should be `${conn(connection-name)}`

Reply:
done